common/lib/xmodule/xmodule/capa_base.py

```diff
@@ -459,6 +459,17 @@ def handle_fatal_lcp_error(self, error):
                 u'Contact course staff in the discussion forum for assistance.</p>'
             )
 
+    def block_is_complete(self):
```

Comment:
Are you using this for verticals too? Might want https://github.com/edx/completion/blob/master/completion/services.py#L102 instead in those cases?

Reply:
So this ended up being removed, but I actually use a [different completion method](https://github.com/edx/edx-platform/blob/master/common/lib/xmodule/xmodule/vertical_block.py#L235) for the vertical case. It's slightly different than the method in the completion service because it takes into account if the child is graded and scored and has weight. The completion service looks at all children.